### ISSUE TITLE
Does mmrazor support instance segmentation model?

### ISSUE THREAD

**Issue by flyzxm5177** [open-mmlab/mmrazor#478]
### Checklist

- I have searched related issues but cannot get the expected help.
- I have read related documents and don't know what to do.

### Describe the question you meet

Such as RTMDet-ins(https://github.com/open-mmlab/mmdetection/tree/3.x/configs/rtmdet)，what should I do if possible, thanks.

**Comment by LKJacky** [comment]
Yes, we support segmentation models. MMRazor works as a model wrapper so that any model can be compressed with MMRazor.

We list some examples for you below.
Distillation
[mmrazor/configs/distill/mmseg/cwd at main · open-mmlab/mmrazor · GitHub](https://github.com/open-mmlab/mmrazor/tree/main/configs/distill/mmseg/cwd)
Pruning
[mmrazor/configs/pruning/base/group_fisher at main · open-mmlab/mmrazor · GitHub](https://github.com/open-mmlab/mmrazor/tree/main/configs/pruning/base/group_fisher)
Quantization
[mmrazor/configs/quantization/ptq/base at main · open-mmlab/mmrazor · GitHub](https://github.com/open-mmlab/mmrazor/tree/main/configs/quantization/ptq/base)

**Comment by IrisDinge** [comment]
Hi, my friends;) 

would like to follow this issuese!
dose mmrazor support mmsegmentation among qat? since ptq could be implemented independendly.
If Yes, could u offer some instances? They could from mmsegmentation for sure.

many thanks!

have a nice day

**Comment by jslok** [comment]
Has anybody been able to apply quantization to rtmdet-ins? Any examples please?

**Comment by Priyanshu88** [comment]
@jslok Are you able to quantize the rtmdet-ins?
